dev/tools/controllerbuilder/scaffold/apis.go

```diff
@@ -139,7 +139,7 @@ func scaffoldDocFile(path string, cArgs *apis.APIArgs) error {
 	if err := WriteToFile(path, out.Bytes()); err != nil {
 		return err
 	}
-	color.HiGreen("New file added %s!\n", path)
+	color.HiGreen("New file added %s\n", path)
```

Comment:
I would suggest we use %q for paths. I don't expect spaces, new lines etc in them but better to be safe.

Reply:
In this case I removed the `!` so that it is clickable.   Agreed though, fixed